### PR TITLE
Add GYM-5025 content

### PR DESCRIPTION
**Pull request by jgagne**
## This PR adds Take 5 GYM-5025 content:

- Front matter: `_data/take5s/GYM-5025.yml`
- Meta stub: `take5/meta/gym-5025.md`
- Transcript: `take5/gym-5025.md`
- Poster artwork: `img/take5/posters/gym-5025.jpg` and `img/take5/posters/gym-5025-hero.jpg`
- OG artwork: `img/take5/og/gym-5025-og.png`
---
- Closes: https://github.com/gymnasium/tracker/issues/19

### CODE REVIEW

File: _data/take5s/GYM-5025.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+course_ID: GYM-5025
+title: "Getting Started With CSS Variables"
+date: 2020-01-27T00:00:00-04:00
+course_type: take5
+url: /static/take5/GYM-5025/
+poster_art: /img/take5/posters/gym-5025.jpg
+live: true
```
Comment (andrewpmiller) [inline _data/take5s/GYM-5025.yml]: Do we want this to go live immediately?

File: _data/take5s/GYM-5025.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+instructor: "Jeremy Osborn"
+topic: "Web Design & Development"
+video_ID: ZLR0rLm_zEM
+video_duration: "4:57"
+featured: true
```
Comment (andrewpmiller) [inline _data/take5s/GYM-5025.yml]: Featured immediately?

**Review by andrewpmiller — CHANGES_REQUESTED** [review]
I think that you meant to keep this unpublished and un-featured. If not, please also add a commit to flip the featured flag on the media query tutorial (to avoid the multiple feature glitch)

File: _data/take5s/GYM-5025.yml (hunk 1 above)
Comment (jgagne) [reply _data/take5s/GYM-5025.yml]: Updated status for production.

File: _data/take5s/GYM-5025.yml (hunk 2 above)
Comment (jgagne) [reply _data/take5s/GYM-5025.yml]: Updated status for production.

**Review by andrewpmiller — APPROVED** [review]
👍

**Pull request merged**